galaxy_skeleton2.R

```diff
@@ -16,7 +17,7 @@ library(tidyverse)    # Contains most of what we need.
 # the file does not end with an "end of line"-character (EOL). This does not
 # seem to pose a problem later, and it seems that we can silece the warning by
 # switchin off the "warn"-argument. Do that if you wish.
-raw_file <- readLines(con = "?")
+raw_file <- readLines(con = "/Users/sondres.b.siger/Documents/BAN400/Assignments/Assignment 5/git-Ciggen/UCNG_Table4.txt")
```

Comment:
You dont need to specify the entire path to the file. Specifying your own path, causes an error if anyone else read ypur script. You can simply set the path to "UCNG_Table4.txt", making sure the working directory is the same as the file.